index.html

```diff
@@ -57,5 +57,57 @@ <h1>DISCOVER THE BEAUTY OF NATURE</h1>
                 
                 </div>
             </div>
+             <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 260">
+      <path
+        fill="rgba(0, 0, 0, 0.6)"
+        fill-opacity="1"
+        d="M0,192L48,181.3C96,171,192,149,288,149.3C384,149,480,171,576,186.7C672,203,768,213,864,218.7C960,224,1056,224,1152,186.7C1248,149,1344,75,1392,37.3L1440,0L1440,320L1392,320C1344,320,1248,320,1152,320C1056,320,960,320,864,320C768,320,672,320,576,320C480,320,384,320,288,320C192,320,96,320,48,320L0,320Z"
+      ></path>
+    </svg>
+
+    <div class="faq">
+      <h3>FAQ'S</h3>
+
+      <div class="questions">
+        <h5 class="q1">Question 1</h5>
+        <p>
+          Vivamus pharetra est vel ultricies auctor. Praesent ipsum justo,
+          porttitor at urna a, consectetur consequat libero. Aenean lacus
+          libero, suscipit nec vulputate rutrum, sodales eu nisl. Fusce
+          fringilla turpis vitae gravida ullamcorper. Etiam semper dui eget
+          ullamcorper elementum. Phasellus dapibus auctor sagittis.
+        </p>
+        <h5 class="q2">Question 2</h5>
+        <p>
+          Nullam pellentesque magna nisl, mollis eleifend dui consectetur a.
+          Nulla congue pellentesque est non iaculis. Suspendisse aliquam ex quis
+          ex lacinia dictum. Duis nulla dolor, dignissim in ultrices in,
+          vehicula nec erat. Duis et rhoncus nulla. Ut mi magna, tempor eget
+          gravida at, imperdiet id enim.Maecenas placerat, lorem eget pharetra
+          accumsan, ligula lorem vulputate ante, vitae vehicula risus erat vel
+          est.
+        </p>
+        <h5 class="q3">Question 3</h5>
+        <p>
+          Vestibulum ante ipsum primis in faucibus orci luctus et ultrices
+          posuere cubilia curae; Sed orci nisi, gravida vel mauris at, convallis
+          volutpat tellus. Donec placerat quis tellus et consectetur. Praesent
+          suscipit porta metus, at ornare lorem blandit at. Vestibulum ante
+          ipsum primis in faucibus orci luctus et ultrices posuere cubilia
+          curae; Donec finibus ullamcorper pretium. Suspendisse dolor risus,
+          mollis vitae velit a, hendrerit vehicula nisl.
+        </p>
+      </div>
+    </div>
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320">
+      <path
+        fill="rgba(0, 0, 0, 0.6)"
+        fill-opacity="1"
+        d="M0,192L48,181.3C96,171,192,149,288,149.3C384,149,480,171,576,186.7C672,203,768,213,864,218.7C960,224,1056,224,1152,186.7C1248,149,1344,75,1392,37.3L1440,0L1440,0L1392,0C1344,0,1248,0,1152,0C1056,0,960,0,864,0C768,0,672,0,576,0C480,0,384,0,288,0C192,0,96,0,48,0L0,0Z"
+      ></path>
+    </svg>
+  </body>
```

Comment:
Can you remove the body and html closing tags from lines 109-110? 
This will create issues in rendering the page as those tags are closing twice.